invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less

```diff
@@ -210,3 +210,12 @@
     display: inline-block;
     float: right;
 }
+
+.exports-list > div.item:first-child {
```

Comment:
I was wondering, if you add the class to the `Semantic UI` theme, [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides#L39), can you avoid the `!important`?
See the other examples there...